ui/src/app/settings/containers/Users/Users.scss

```diff
@@ -0,0 +1,29 @@
+@import "src/scss/vanilla";
+
+.user-list__button.p-button--base {
+  // Stop the icons from adding extra height to the row:
+  margin-top: -0.25rem;
+  margin-bottom: -0.25rem;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+}
+
+.user-list .p-table-expanding__panel {
+  padding-top: 0;
+}
+
+.p-table-expanding th.user-list__cell--small,
+.p-table-expanding td.user-list__cell--small {
```

Comment:
Could this modifier be added to the table-expanding pattern scss. If someone needs to modify this styles the first place they will look will be the pattern file.

Reply:
Really I think we need more control over the column widths than this from looking at the design, do you have any suggestions about a good way to set the width of each column?

Reply:
Oops, you answered below.